kanban/templates/email_templates/assignee_change_notification.html

```diff
@@ -0,0 +1,13 @@
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <title>Unfinished task ran out of time</title>
+</head>
+
+<body>
+    <p> Hello, {{ context.assignee }}!</p>
+    <p> Issue: {{ context.issue_title }} passed from {{ context.previous_assignee }} to {{ context.assignee }}. </p>
+</body>
+
+</html>
```

Comment:
missing empty line on end of file

Reply:
It's your black settings that disallowed me to add an empty line there!